https://github.com/c-ares/c-ares/commit/eebfe0c15e1cc231500dfd79c730788790ec5251

From eebfe0c15e1cc231500dfd79c730788790ec5251 Mon Sep 17 00:00:00 2001
From: Brad House <brad@brad-house.com>
Date: Sun, 17 Dec 2023 19:03:33 -0500
Subject: [PATCH] Fix bad stub for ares__iface_ips_enumerate()

If the ability to enumerate interface ip addresses does not exist
on a system, the stub function contained the wrong prototype.

Fixes Bug: #668
Fix By: Brad House (@bradh352)
---
 src/lib/ares__iface_ips.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/lib/ares__iface_ips.c b/src/lib/ares__iface_ips.c
index e10a0d80c..7cd12a605 100644
--- a/src/lib/ares__iface_ips.c
+++ b/src/lib/ares__iface_ips.c
@@ -510,11 +510,10 @@ static ares_status_t ares__iface_ips_enumerate(ares__iface_ips_t *ips,
 #else
 
 static ares_status_t ares__iface_ips_enumerate(ares__iface_ips_t     *ips,
-                                               ares__iface_ip_flags_t flags,
                                                const char            *name)
 {
   (void)ips;
-  (void)flags;
+  (void)name;
   return ARES_ENOTIMP;
 }
 

